activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb

```diff
@@ -367,6 +367,17 @@ def primary_keys(table_name) # :nodoc:
           SQL
         end
 
+        def primary_key_name(table_name) # :nodoc:
```

Comment:
I'm not sure about adding this new method to connection just for testing purposes, can we execute this query in the test instead?